system_tests/espresso_transaction_payload_signature_test.go

```diff
@@ -0,0 +1,70 @@
+package arbtest
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/offchainlabs/nitro/arbos"
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"testing"
+)
+
+func TestEspressoTransactionSignatureForSovereignSequencer(t *testing.T) {
```

Comment:
I would like to make this test simpler, but so far I don't have any clue

Reply:
@ImJeremyHe @Sneh1999 based on the discussion yesterday. We will likely implement an Espresso pre-confirmation node that finalizes L2 blocks when Espresso finalized them. If we are still on sovereign sequencer v1 at that time let's implement this signing and test it. If things go well and we have the v2 we don't need to bother with the signing. So I would suggest we keep this PR open but stop working on it and don't merge it right now then finish it later if necessary, or hopefully just close it.